### PR TITLE
Feature/refactor anthony

### PR DESCRIPTION
**Pull request by studioant**
### Refactor T1 code
Applied refactoring to multiple classes to provide greater uniformity

### CODE REVIEW

**Comment by studioant** [comment]
Looks like some conflicts have occurred after previous merge. I will resolve these shortly

**Comment by studioant** [comment]
@Delcari, merge conflicts resolved, ready for re-review

**Review by Delcari — CHANGES_REQUESTED** [review]
Seems you reverted a number of changes I had just merged into the main branch from Feature/refactor #30.
I think it'd be wise to revert the last commit and try again! 😞 
I'm happy to perform the merge/resolve conflicts if needed, cheers.

**Comment by studioant** [comment]
ah, I neglected to commit the merged copy. should be ok now

File: Helper.h [hunk 1]
```diff
@@ -1,171 +1,205 @@
...
+        }
+
+        /**
+         * @brief Find the config files in the games directory.
+         * 
+         * @param dir The games directory
+         * @return * vector<string> 
+         */
+        vector<string> get_config_files(string dir)
+        {
```
Comment (Delcari) [inline Helper.h:50]: Still some changes that have been removed it seems? Haven't had a look over all the changes but this function has been affected 😢

**Review by lachfoy — COMMENTED** [review]
I'm not sure I understand the triple forward slash (///) for some comments, but otherwise everything looks good!

**Comment by studioant** [comment]
> I'm not sure I understand the triple forward slash (///) for some comments, but otherwise everything looks good!

/// indicates a short description that will be added to the Doxygen documentation. // will not add anything to Doxygen documentation.

**Review by Delcari — APPROVED** [review]
Just glossed over it again, all clear 😄..
Merge whenever ☑️

Probably may as well rename the other classes here too
### EG.
```'gridlayout.h' -> GridLayout.h'```

**Pull request merged**